### ISSUE TITLE
 Commands like "Block" and "addtexturetoterrain" not working?

### ISSUE THREAD

**Issue by Hazard-Hat** [Turnip-Labs/bta-example-mod#2]
For some reason the commands aren't working like they should.
![Screenshot (870)](https://user-images.githubusercontent.com/96910731/202832389-24740ca9-5f73-42e1-bb24-036d68e67853.png)

But in the minimal_mod that does have the examples, do work?
![Screenshot (871)](https://user-images.githubusercontent.com/96910731/202832435-5480da64-5261-4a6b-97f1-9c89335fbf42.png)

Idk what I'm missing, do I need to add executions for the commands to work?

**Comment by azuredao8** [comment]
Sorry for not seeing this issue earlier. It appears that there's something wrong with your IDE because even vanilla Minecraft classes like Block or Material are highlighted in red. Please tell me whether this issue is still occuring or whether you've fixed it already.

**Comment by Hazard-Hat** [comment]
> Sorry for not seeing this issue earlier. It appears that there's something wrong with your IDE because even vanilla Minecraft classes like Block or Material are highlighted in red. Please tell me whether this issue is still occuring or whether you've fixed it already.

I'm currently using "Intellig IDEA Community Edition 2022.2.3" and have not solved the problem, I have no clue.
I decided to download it again and start over, opened it, and just pasted both of the commands (not the texture PNG though)
And I did find out that "Add Texture" wasn't working because I forgot the curly bracket, but it still seems to throw a fit with something?
I added the curley on the old one, but it still throws a fit with Block Helper, but the current one has this error, can't find any answers online as the line seems to be a 1:1 with how it is in the example-demo?

![Screenshot (1287)](https://user-images.githubusercontent.com/96910731/207245459-3e9177f5-ed60-4e22-9d2e-1320e25bc856.png)
